mender-deb-package

```diff
@@ -189,13 +193,50 @@ build_packages() {
     # dh_make returns 1 because debian directory exists,
     # hence the || true, without orig file buildpackage command would fail
     dh_make --file /tmp/"${DEB_PACKAGE}_${DEB_VERSION_NO_REV}".orig.tar.gz --packagename "${DEB_PACKAGE}_${DEB_VERSION_NO_REV}" -s -y || true
+
+  fi
+
+  dpkg_build_ignore_flag="--diff-ignore=(.git|vendor)"
+
+
+  skip_patches_flag="--source-option=--skip-patches"
+  if [ "$OS_DISTRO" = "debian" -a "$OS_CODENAME" = "buster" ]; then
+    echo "deb http://deb.debian.org/debian buster-backports main" > /etc/apt/sources.list.d/buster-backports_deps.list
+    cat <<EOF> /etc/apt/preferences.d/prefer_backports.pref
+Package: cmake*
+Pin: release a=buster-backports
+Pin-Priority: 999
+EOF
+    apt update
+    # We want the patches applied for Debian Buster
+    rm -f debian/source/local-options
+    skip_patches_flag=
+  else
+    # Skip applying the patches
+    rm -f debian/patches/series
   fi
 
+  # Install the builder architecture independent dependencies from the debian/control file
+  mk-build-deps --install --build-indep --tool='apt-get -o Debug::pkgProblemResolver=yes --no-install-recommends --yes' debian/control
+
+  # HOST - the machine arch we are building _for_
+  HOST_ARCH=${ARCH}
+
+  dpkg --add-architecture ${ARCH}
+  apt update
```

Comment:
This we are doing already in the Dockerfile (except for the armhf on Debian special handling which I think we should remove in this PR)